samples/caf_sensor_manager/src/main.c

```diff
@@ -3,20 +3,43 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
-#include <app_event_manager.h>
+
+#include <zephyr.h>
 
 #define MODULE main
-#include <caf/events/module_state_event.h>
 
+#include <app_event_manager.h>
+
+#include <caf/events/module_state_event.h>
 #include <logging/log.h>
+
 LOG_MODULE_REGISTER(MODULE);
 
+#if CONFIG_BOARD_ENABLE_CPUNET
+#include "event_proxy_init.h"
+#endif
 
 void main(void)
 {
-	if (app_event_manager_init()) {
-		LOG_ERR("Event manager not initialized");
-	} else {
-		module_set_state(MODULE_STATE_READY);
+	int ret;
+
+	ret = app_event_manager_init();
+	if (ret) {
+		LOG_ERR("Event Manager not initialized, err: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
+	}
+	LOG_INF("Event manager initialized");
+
+
+#if CONFIG_BOARD_ENABLE_CPUNET
```

Comment:
why do we care if CPUNET is enabled. Simply check if event manager proxy is enabled (which should be in prj.conf of this board).